toolkit/app-write-mram.py

```diff
@@ -196,15 +196,18 @@ def main():
     parser.add_argument(
         "-V", "--version", help="Display Version Number", action="store_true"
     )
+    parser.add_argument("--cfg-part", type=str, help="Part Number")
+    parser.add_argument("--cfg-rev", type=str, help="Part Revision", default="B4")
+    parser.add_argument("--cfg-jtag", type=str, help="JTAG Interface", default="J-Link")
+    parser.add_argument("--cfg-mram", type=str, help="MRAM Interface", default="isp")
     parser.add_argument("-v", "--verbose", help="verbosity mode", action="store_true")
-
     args = parser.parse_args()
     if args.version:
         print(TOOL_VERSION)
         sys.exit()
 
     # memory defines for Alif/OEM MRAM Addresses and Sizes
-    load_global_config()
+    load_global_config(args.part, args.rev, args.jtag, args.mram)
```

Comment:
These arg names must change to match, eg `args.cfg_part`.

Reply:
Sorry, fixed.